docker-compose.yml

```diff
@@ -121,12 +123,11 @@ services:
         image: 'redis:3.2.7'
     article-store-airflow-scheduler:
         build:
-          context: ./workflow/airflow_libero
+          context: ./airflow
         depends_on:
             - article-store-airflow-webserver
         volumes:
-            - ./workflow/airflow_libero/dags:/usr/local/airflow/dags
-            - ./workflow/airflow_libero/plugins:/usr/local/airflow/plugins
```

Comment:
I was getting an import error:

`ImportError: cannot import name 'EventEmittingPythonOperator`

It's because the `plugins` volume is still needed. I believe only the scheduler and the worker need it.

Reply:
It should be in the container...

Reply:
Do you have an old `__pycache__` in the `dags` folder that's being mounted?

Reply:
There was no `__pycache__` in the mounted `dags` folder but after a `down -v` and a fresh build it's working for me 👍